res/values/strings.xml

```diff
@@ -544,8 +544,8 @@
     <string name="qrshow_join_group_hint">Scan this to join the group \"%1$s\".</string>
     <string name="qrshow_join_contact_title">QR invite code</string>
     <string name="qrshow_join_contact_hint">Scan this to set up a contact with %1$s</string>
-    <string name="qrshow_join_contact_no_connection_hint">QR code setup requires an internet connection. Please connect to a network before proceeding.</string>
-    <string name="qrshow_join_contact_no_connection_toast">No internet connection, can\'t perform QR code setup.</string>
+    <string name="qrshow_join_contact_no_connection_hint">Please connect to the Internet before proceeding.</string>
+    <string name="qrshow_join_contact_no_connection_toast">Connect to the Internet to perform QR code setup.</string>
```

Comment:
to set up ?? by scanning the QR code?

Reply:
not by scanning the QR code, but yes the setup requires an internet connection :)